### PR TITLE
Remove duplicated words in documentation

### PR DESCRIPTION
**Pull request by rabadin**

### CODE REVIEW

**Comment by rabadin** [comment]
@mattfarina Thanks for the review. I've made the requested changes.

File: incubator/webpagetest-server/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: webpagetest-server
-version: 0.2.0
-appVersion: "2018-03-08"
+version: 0.2.1
+appVersion: "2018-10-30"
```
Comment (unguiculus) [inline incubator/webpagetest-server/Chart.yaml]: Why this change? It's unrelated. Also, readme and values.yaml still have `2018-03-08`.

Comment (rabadin) [reply incubator/webpagetest-server/Chart.yaml]: Ah yes, you're right, my bad. Fixed.
